### ISSUE TITLE
Duplication of Effort?

### ISSUE THREAD

**Issue by hamiltoncj** [stefano998/geodesic2shape#2]
It looks like what you have may be a duplication of  "Shape Tools" functionality. It would be nice to work together rather than have separate efforts. Incidentally, I think the statement "Vicenty formulae provides sub-millimeter accuracy" is somewhat misleading. It provides that accuracy on the ellipsoid without elevation taken into account. Surface measurements will be different. As far as I know geographiclib has the most accurate geodesic algorithms and there are some issues with the Vincenty formula where it does not converge to a solution under certain situations.

**Comment by stefano998** [comment]
Hey, thanks for your feedback!
- It looks like what you have may be a duplication of "Shape Tools" functionality (You are right, that is a good practice I shall consider in next versions);
- I think the statement "..." is somewhat misleading (Ok, I have just changed the statement. I think it is better now. Thank you for the alert); 
- geographiclib has the most accurate geodesic algorithms (Yes, but Vicenty is also very accurate: sub-millimeter accuracy, what is more than enough for most applications);  
- Vincenty formula does not converge to a solution under certain situations (Only the inverse problem with 2 antipodal points does not converge. If this unlikely case happens, the plugin warns the user);
- In conclusion, I agree that geographiclib seems to be more accurate, but I want to highlight that Vicenty is still very accurate (sub-millimeter accuracy). Moreover, as far as I know, geographiclib has no tool to export the shapefile (SHP) of a respective segment of a geodesic's line, what is the main point of the plugin (Geodesic2shape).
